lottie-swift/src/Public/iOS/AnimatedSwitch.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  AnimatedSwitch.swift
+//  lottie-swift
+//
+//  Created by Brandon Withrow on 2/4/19.
+//
+
+import Foundation
+import UIKit
+/**
+ An interactive switch with an 'On' and 'Off' state. When the user taps on the
+ switch the state is toggled and the appropriate animation is played.
+ 
+ Both the 'On' and 'Off' have an animation play range associated with their state.
+ */
+final public class AnimatedSwitch: AnimatedControl {
+  
+  /// The current state of the switch.
+  public var isOn: Bool {
+    set {
```

Comment:
I don't readily see the reason this was implemented with separate `isOn` and `_isOn` variables. Unless I'm mistaken, it appears it can be consolidated into one as the first line of `updateOnState(isOn:, animated:)` sets the values to match anyway. Maybe simpler to use a `didSet` here?

Reply:
Its a hold over from an earlier version. Can be refactored.